docs/user-docs/latest/plugins/installing-plugins-via-marketplace.md

```diff
@@ -4,37 +4,41 @@ description: Instructions to install new plugins in a Strapi application via the
 canonicalUrl: https://docs.strapi.io/user-docs/latest/plugins/installing-plugins-via-marketplace.html
 ---
 
-# Installing plugins via the Marketplace
+# Marketplace plugins
 
-::: callout 🚧 The Marketplace is currently not available for v4. It will be back soon, stay tuned!
-In the meantime, plugins can be installed via the Command Line Interface (see [Developer Documentation](/developer-docs/latest/developer-resources/cli/CLI.md#strapi-install)).
-:::
-
-The Marketplace is a section of the admin panel that lists all plugins that can be installed in a Strapi application. The Marketplace is accessible from ![Marketplace icon](../assets/icons/marketplace.svg) _Marketplace_ in the main navigation of the admin panel.
+The Marketplace contains optional plugins built to customize the core Strapi installation. The Marketplace is located in the admin panel, indicated by the ![Marketplace icon](../assets/icons/marketplace.svg) _Marketplace_. Marketplace plugins are installed using the CLI (see [Developer Documentation](/developer-docs/latest/developer-resources/cli/CLI.md#strapi-install)).
 
-<!--
+::: tip
+The Marketplace displays only v4 plugins. Both v3 and v4 plugins are discoverable in the [Strapi Market](https://market.strapi.io/). v3 and v4 plugins are not cross-compatible.
```

Comment:
Mentioning the Strapi Market and the fact that the in-app Marketplace only displays v4 plugins: that's a note.
But stating that v3 & v4 apps/plugins are not compatible, that's more a caution I believe, so I wouldn't put these different pieces of information together, otherwise people may overlook the most important part (the caution).